api/secret.go

```diff
@@ -189,6 +172,23 @@ DONE:
 	return policies, nil
 }
 
+// TokenAccessors returns the standardized list of accessors for the given secret
+func (s *Secret) TokenAccessors() ([]string, error) {
+	if s == nil {
+		return nil, nil
+	}
+
+	if s.Auth != nil && len(s.Auth.Policies) > 0 {
+		return s.Auth.Policies, nil
```

Comment:
This doesn't look correct.  Policies aren't accessors, right?

Reply:
Good catch. fixed now